vortex-array/src/lib.rs

```diff
@@ -60,3 +65,15 @@ pub mod flatbuffers {
     //! Re-exported autogenerated code from the core Vortex flatbuffer definitions.
     pub use vortex_flatbuffers::array::*;
 }
+
+static USE_VORTEX_OPERATORS: LazyLock<bool> = LazyLock::new(|| {
+    std::env::var("VORTEX_OPERATORS")
+        .map(|v| v == "1" || v.to_lowercase() == "true")
+        .unwrap_or(false)
+});
```

Comment:
I think you can just do this, so it gets turned on [as long as it is set](https://doc.rust-lang.org/std/env/fn.var.html):

```suggestion
static USE_VORTEX_OPERATORS: LazyLock<bool> = LazyLock::new(|| {
    std::env::var("VORTEX_OPERATORS").is_ok()
});
```

Reply:
I always find it confusing if MY_ENV=false turns the thing on 🤷‍♂️